docs/semgrep-secrets/policies.md

```diff
@@ -0,0 +1,119 @@
+---
+slug: policies
+append_help_link: true
+title: Policies
+hide_title: true
+description: The Policies page is a visual representation of the rules that Semgrep Secrets uses to scan code
+tags:
+  - Semgrep Secrets
+---
+
+# Policies
+
+The **Policies** page visually represents the rules Semgrep Secrets uses for scanning. 
+
+![Overview of Semgrep Secrets policies view](/img/secrets-rules-management.png)
+
+To access the policies page for Semgrep Secrets:
+
+1. Log in to Semgrep Cloud Platform and navigate to **Rules** > **Policies**.
+2. Switch to the **Secrets** page.
+
+## Policies page structure
+
+The Policies page consists of a header and three main panes:
+
+<dl>
+    <dt>Policies header</dt>
+        <dd>
+            The top header consists of <strong>Validation State Policies</strong>, which lets you define how Semgrep handles findings that it categorizes as invalid or results in a validation error
+        </dd>
+    <dt>Filter pane</dt>
+        <dd>
+           Displays filters to select and perform operations on rules in bulk quickly. See <a href="#filters">Filters</a> for more information.
+        </dd>
+    <dt>Rules pane</dt>
+        <dd>
+            The rules pane displays the rules that Semgrep scans use to detect findings and allows you to edit their assigned rule modes. You can make these edits either one by one or through bulk editing of many rules. You can also use the <strong>Search for rule names or ids</strong> box. See <a href="#filters">Filters</a> for more information.
+        </dd>
+</dl>
+
+## Filters
+
+The filter pane displays filters to select and perform operations on rules in bulk. The following filters are available:
+
+| Filter | Description |
+| - | - |
+| Modes | Filter by the workflow action Semgrep performs when a rule detects a finding. An additional filter, **Disabled**, is provided for rules you have turned off and are no longer included for scanning. |
+| Validation | Filter by whether the rule includes a validator or not. |
+| Type | Filter by the type of secret the rule addresses. Examples: AWS, Adobe, DigitalOcean, GitHub, GitLab. |
+| Source | Filter by Pro secrets (authored by Semgrep) or by Custom secrets (authored by users) |
+| Severities | Filter by the severity level of the secret |
```

Comment:
Do we have any docs where we state how we decide on severity levels for secrets?